airflow/cli/cli_parser.py

```diff
@@ -1359,8 +1359,20 @@ class GroupCommand(NamedTuple):
     ActionCommand(
         name='upgrade',
         help="Upgrade the metadata database to latest version",
+        description=(
+            "Upgrade the schema of the metadata database. "
+            "To print but not execute commands, use option `--sql-only`. "
+            "If using options `--from-revision` or `--from-version`, you must also use `--sql-only`, "
```

Comment:
since this are RST we need double backticks
```suggestion
            "If using options ``--from-revision`` or ``--from-version``, you must also use ``--sql-only``, "
```